.github/workflows/security.yml

```diff
@@ -21,15 +21,25 @@ jobs:
         run: |
           echo "NOW=$(date +'%Y-%m-%d')" >> "$GITHUB_OUTPUT"
 
-      - name: Checkout Main Branch for .zap/rules.tsv
+      - name: Checkout Repo for .zap/rules.tsv
         uses: actions/checkout@v4
         with:
-          ref: 'main'
+          ref: main
```

Comment:
It looks like we'd want to remove this so that the action checks out the ref associated with the workflow run. I'll take care of this in a subsequent PR.